plugins/rtmp-services/data/services.json

```diff
@@ -3408,13 +3408,16 @@
             }
         },
         {
-            "name": "Dolby Millicast",
+            "name": "Dolby OptiView Real-time",
```

Comment:
Is it actually referred to as "Dolby OptiView Real-time" anywhere? I can only find references to "Dolby OptiView".